notebooks/In-processing.ipynb

```diff
@@ -17,17 +17,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
     "# @hidden_cell\n",
     "# The following code contains the credentials for a file in your IBM Cloud Object Storage.\n",
     "# You might want to remove those credentials before you share your notebook.\n",
     "credentials = {\n",
-    "    'IAM_SERVICE_ID': 'iam-ServiceId-574c8012-6175-4db4-826c-8951908d9e1e',\n",
-    "    'IBM_API_KEY_ID': 'mecmxiDlkkNBAx_xwFoWofXDFgfjwrYliwcdLryqmvu7',\n",
+    "    'IAM_SERVICE_ID': 'IAM_SERVICE_ID',\n",
+    "    'IBM_API_KEY_ID': 'IBM_API_KEY_ID',\n",
```

Comment:
As much as I disagree with checking in keys it seems this is needed to download the `Pipeline_LabelEncoder-0.1.zip` library. We should consider making it publicly available.

```suggestion
    "    'IAM_SERVICE_ID': 'iam-ServiceId-574c8012-6175-4db4-826c-8951908d9e1e',\n",
    "    'IBM_API_KEY_ID': 'mecmxiDlkkNBAx_xwFoWofXDFgfjwrYliwcdLryqmvu7',\n",
```